optimum/neuron/modeling_traced.py

```diff
@@ -176,6 +192,7 @@ def _from_pretrained(
         local_files_only: bool = False,
         model_save_dir: str | Path | TemporaryDirectory | None = None,
         neuron_config: "NeuronDefaultConfig | None" = None,
+        cpu_backend: bool = False,
```

Comment:
`_from_pretrained` is for loading the checkpoiunt, it would be meaningful only on neuron devices in our case. So instead if overriding the func which should never be called if there is no neuron equipped, you can just skip loading the exported artifacts in `_export`.